openedx/core/djangoapps/user_api/views.py

```diff
@@ -449,11 +449,12 @@ def _add_level_of_education_field(self, form_desc, required=True):
         # form used to select the user's highest completed level of education.
         education_level_label = _(u"Highest level of education completed")
 
+        options = [(name, _(label)) for name, label in UserProfile.LEVEL_OF_EDUCATION_CHOICES]  # pylint: disable=translation-of-non-string
```

Comment:
@mtyaka Since this makes the strings non-extractable, I had to go look at https://github.com/edx/edx-platform/blob/69c47e09511c0fda4762dc959a6a3025dc96f272/common/djangoapps/student/models.py#L252-L264 to realize that the original string definition was marking the strings for extraction but not translating them via `ugettext_noop()` - I'm guessing this is because the translation would then happen too early, but it would be good to have a comment about this somewhere. (And same thing for genders)